test/package_config_info_test.dart

```diff
@@ -157,7 +160,7 @@ void main() {
 
     test("with a matching package", () {
       expect(resolver.packagePath("foo"), equals(p.current));
-    });
+    }, onPlatform: {'!vm': new Skip()});
```

Comment:
You should never commit a skip without a description.

If this test isn't compatible with the browser, you should use `testOn: "vm"`, not a skip. Skips are for tests that are broken and need to be fixed.

Same below.